src/Transition_Systems-Weak_Transition_Systems.thy

```diff
@@ -284,6 +284,30 @@ proof -
     by simp
 qed
 
+lemma weak_tau_transitions_up_to_weak_bisimilarity_rule:
+  assumes
+    "p \<Rightarrow>\<lparr>\<tau>\<rparr> p'"
+  and
+    "p' \<approx> q"
+  and
+    "q \<Rightarrow>\<lparr>\<tau>\<rparr> q'"
+  and
+    "q' \<approx> p"
```

Comment:
All the assumptions should go on a single line, as they easily fit into it.

Reply:
Fixed in 5de722f7ae50fa60ce0a18d217f8db1c0e71f59d, please verify.

Reply:
Nice.